src/main/resources/static/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Welcome to SurveyMonkey</title>
+</head>
+<body>
+    <h1 th:text="${message}">Welcome!</h1>
+    <a th:href="@{/createSurvey}" class="btn btn-primary"><button>Create survey</button></a>
+    <a th:href="@{/addRemoveQuestions}" class="btn btn-primary"><button>View survey</button></a>
```

Comment:
What survey is this supposed to link to? If this was working it would cause an error since no survey ID is provided in the URL.